src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -1,21 +1,29 @@
-﻿@using SFA.DAS.CommitmentsV2.Shared.Extensions
+﻿@using SFA.DAS.ProviderUrlHelper
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
 @using SFA.DAS.CommitmentsV2.Types
 @model SFA.DAS.ProviderCommitments.Web.Models.Apprentice.DetailsViewModel
+@inject ILinkGenerator LinkGenerator
 
 @{
     ViewData["Title"] = "Apprentice Details";
 }
 
-
-<h1 class="govuk-heading-xl">@Model.ApprenticeName</h1>
-
-
 @*https://localhost:5001/10005077/apprentices/4MBBNM*@
 
 <div class="govuk-grid-row govuk-!-margin-bottom-2">
     <div class="govuk-grid-column-two-thirds">
-        <h2 class="govuk-heading-m">Apprentice status</h2>
 
+        @if (Model.PendingUpdate)
+        {
+            <div class="das-notification" tabindex="-1">
+                <p class="das-notification__body">There are changes to this apprentice's details that are waiting for approval by the employer.</p>
+                <p class="das-notification__body"><a href="@LinkGenerator.ProviderApprenticeshipServiceLink($"{Model.ProviderId}/apprentices/manage/{Model.ApprenticeshipHashedId}/undo")" class="govuk-link">View changes</a> </p>
```

Comment:
There's a `ProviderApprenticeshipServiceLink` Url helper method we could use instead (see Edit link further down)